sg_covid_impact/sic.py

```diff
@@ -1,43 +1,59 @@
 import requests
 import os
 import logging
-import sg_covid_impact
 import re
+import json
 import pandas as pd
 
+# from toolz.curried import pipe
+
+import sg_covid_impact
+
 project_dir = sg_covid_impact.project_dir
 
+_SIC_OUTPUT_FILE = f"{project_dir}/data/raw/sic_2007.xls"
+_SIC_LOOKUP_FILE = f"{project_dir}/data/raw/lookup.json"
```

Comment:
`sic_division_lookup.json`  helps differentiate this from other lookups and other SIC lookups

Reply:
It probably belongs in `processed` too